index.js

```diff
@@ -305,252 +301,89 @@ app.get('/slack', function(req, res) {
+});
 
-renderer.code = function(code, lang) {
-  if (lang == 'js') {
-    return '<pre><code class="lang-js"><div style="line-height:1.1em;">' + require('highlight.js').highlightAuto(code).value + '</div></code></pre>';
-  } else {
-    return '<code><cli>' + code + '</cli></code>';
-  }
-};
+app.get(originalBase + 'hardwareAPI', function (req, res) {
+  res.redirect(gitbooksBase + 'API/Hardware_API');
+});
 
-app.get('/docs/:slug', function(req, res) {
-  var doc = req.params.slug;
-  if (!rawDocs[doc]) {
-    return res.status(404).send("Sorry page not found");
-  }
-  var forkPath = uri.parse(rawDocs[doc].url).pathname.split('/');
-  var forkUrl = '//github.com/' + forkPath[1] + '/' + forkPath[2];
-  // update on anything older than 1 hour
-  var d = new Date((new Date()) * 1 - 1000 * 3600 * 1);
-  // if any urls have blank code, request it
-  if ((rawDocs[doc].text === '' && rawDocs[doc].url) || (rawDocs[doc].updated < d.valueOf())) {
-    request.get(rawDocs[doc].url, function(err, data) {
-      if (err) {
-        return console.error('could not get', doc, rawDocs[doc].text, err);
-      }
-      // Turn markdown into html
-      headingList = [];
-      var text = marked(data.body, {
-        renderer: renderer
-      });
-      rawDocs[doc].heading = headingList;
-      // Replace internal links referring gh docs with links referring to imported docs
-      var myURL = rawDocs[doc].url.replace('/raw.githubusercontent.com/', '/github.com/').replace('/master/', '/blob/master/');
-      text = replaceAll(myURL, rawDocs[doc].newlink, text);
-
-      rawDocs[doc].text = text;
-      rawDocs[doc].updated = Date.now();
-      // console.log('doc', doc, headingList);
-      res.render('doc', {
-        navbar: indexdata.navbar,
-        link: forkUrl,
-        sideBar: sideBarLists,
-        expandedSide: rawDocs[doc].heading,
-        title: doc,
-        text: rawDocs[doc].text
-      });
-    });
-  } else {
-    res.render('doc', {
-      navbar: indexdata.navbar,
-      link: forkUrl,
-      sideBar: sideBarLists,
-      expandedSide: rawDocs[doc].heading,
-      title: doc,
-      text: rawDocs[doc].text
-    });
-  }
+app.get(originalBase + 'networkAPI', function (req, res) {
+  res.redirect(gitbooksBase + 'API/Network_API');
+});
+
+app.get(originalBase + 'modules', function (req, res) {
+  res.redirect(gitbooksBase + 'API/Modules');
+});
+
+app.get(originalBase + 'cli', function (req, res) {
+  res.redirect(gitbooksBase + 'API/CLI');
+});
+
+app.get(originalBase + 'compatibility', function (req, res) {
+  res.redirect(gitbooksBase + 'API/Languages');
+});
+
+app.get(originalBase + 'communicationProtocols', function (req, res) {
+  res.redirect(gitbooksBase + 'Tutorials/Communication_Protocols');
+});
+
+app.get(originalBase + 'DIYModule', function (req, res) {
+  res.redirect(gitbooksBase + 'Tutorials/Making_Your_Own_Module');
+});
+
+// Even weirder legacy redirects to places that aren't found in the gitbook version of T2 docs
```

Comment:
@johnnyman727 @tcr @HipsterBrown you all might have opinions on this.

We no longer have a "docs" page that points to all of our source files, since it's linked in other places. We also don't pull in the docs for each tenpin module in the new version of gitbooks. Any thoughts/recommendations for better ways to redirect these pages?